core/src/SYCL/Kokkos_SYCL_Instance.cpp

```diff
@@ -62,14 +63,8 @@ SYCLInternal::~SYCLInternal() {
   }
 }
 
-int SYCLInternal::verify_is_initialized(const char* const label) const {
-  if (!is_initialized()) {
-    Kokkos::abort((std::string("Kokkos::Experimental::SYCL::") + label +
-                   " : ERROR device not initialized\n")
-                      .c_str());
-  }
-  return is_initialized();
-}
+KOKKOS_IMPL_SPACE_VERIFY_IS_INITIALIZED(SYCL, !is_initialized(), is_initialized())
```

Comment:
Todo for me: it seems this one needs to add `::Experimental::` in the error message.